### ISSUE TITLE
Tekst ved innlogging som man må godta

### ISSUE THREAD

**Issue by mro046** [Rapporteket/norspis#15]
Legge til pop-up-tekst ved innlogging som bruker må godkjenne. 

I gamle Rapporteket ble følgende tekst laget til NorSpis og kom opp ved innlogging:

ADVARSEL
Vi anmoder om at rapportene i Rapporteket brukes kritisk. En viktig feilkilde vil trolig være ufullstendige datasett, som gjør at man særlig må vurdere representativiteten kritisk. Lav dekningsgrad, samt at ikke alle data er ferdigstilt for aktuell tidsperiode, vil bidra til dårlig representativitet. Rapportene er ment for intern bruk ved de registrerende behandlingsenhetene. Rapportene inneholder anonymiserte data. I tilfeller der man foretar utvalg som gir rapporter med få pasienter og hvor man kombinerer flere ulike utvalg og rapporter, kan det ikke utelukkes med absolutt sikkerhet at rapporterte data kan tilbakeføres til enkeltpersoner. Av de ovennevnte grunnene ber NorSpis om at rapportene dere henter ut fra Rapporteket ikke blir brukt i offentlige sammenhenger eller offentliggjort på noe vis.

Den samme må legges inn nå, men registeret bør i test gå over den på nytt og godkjenne (siden det er lenge siden denne er laget).

**Comment by areedv** [comment]
Dette henger ikke helt sammen, og det kommer jeg tilbake til. Rapporteket kan av prinsipp og ved design ikke drive med adgangskontroll (ei heller identitetssjekk). Rapporteket tar imot brukere som har vært gjennom slik kontroll hos helseregister.no (eller annen tjeneste som sørger for autentisering/autorisasjon).

Så, den forutsetningen som her settes opp må påtvinges ved søknad om tilgang til Rapporteket for NorSpis og ikke etter at man har fått adgang. Det er dere/NorSpis som skal godkjenne søknader om tilgang til Rapporteket og en slik "disclaimer" må legges til den delen av prosessen

**Comment by mro046** [comment]
Ja, ved nærmere ettersyn ser jeg at det nok stemmer at advarselen er tilknyttet Helseregister og ikke Jaspersoft. Vil derfor tro advarselen fortsatt vil kunne poppe opp som før/ slik den gjør i dag, inne på Helseregister.no. Jeg kan forhøre meg med HN-IKT.

**Comment by areedv** [comment]
Bare for å presisere da: det kan godt finnes en info-tekst på Rapporteket (som om ønskelig kommer opp når man begynner en økt), men det er ikke aktuelt å nekte tilgang for brukere som allerede er "akkreditert". Det naturlige da er å legge slik informasjon på ei "framside"

**Comment by mro046** [comment]
Forstår. Den ble aldri laget for å nekte tilgang, da de som du også sier, allerede er innlogget på Helseregister og slik sette allerede inne, men teksten ble laget som en tekst man måtte velge "jeg forstår" for å komme videre (altså kun et mindre risikoreduserende tiltak, som kommer i tillegg til anne, for å forhindre feil bruk av rapporter).

Vi ser hva HN-IKT svarer på-post jeg har sendt dem. Hvis det ivaretas av dem inne på Helseregister.no,før videresending til Rapporteket,  er det ikke behov for å gjøre noe videre i denne saken.

**Comment by areedv** [comment]
tullekommentar
